sechub-shared-kernel/src/test/java/com/mercedesbenz/sechub/sharedkernel/messaging/MapStringToListOfStringsMessageDataProviderTest.java

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.sharedkernel.messaging;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+class MapStringToListOfStringsMessageDataProviderTest {
+
+    private MapStringToListOfStringsMessageDataProvider providerToTest = new MapStringToListOfStringsMessageDataProvider();
```

Comment:
Same as for ListOfStringsMessageDataproviderTest:

please divide tests - we want only ONE executed part per test, so it is clear WHAT is tested

Also `""` is not handled inside the tests as well